apps/interpolate/CMakeLists.txt

```diff
@@ -1,4 +0,0 @@
-halide_project(interpolate "apps" interpolate.cpp)
```

Comment:
Can we have CMake support for this one too?

Reply:
Oops, forgot to commit the new version of it. Done.